### PR TITLE
Added CLion image

### PR DESCRIPTION
**Pull request by ntimo**
This MR adds the Jetbrains clion image since clion is supported by coder and there has not been an image for it yet.

Sources: 
- https://coder.com/docs/coder/v1.19/workspaces/editors#jetbrains-ides-in-the-browser
- https://www.jetbrains.com/de-de/clion/

### CODE REVIEW

File: images/clion/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Enterprise GoLand
+
+[![Docker Pulls](https://img.shields.io/docker/pulls/codercom/enterprise-goland?label=codercom%2Fenterprise-goland)](https://hub.docker.com/r/codercom/enterprise-goland)
```
Comment (bpmct) [inline images/clion/README.md:3]:
```suggestion
# Enterprise CLion

[![Docker Pulls](https://img.shields.io/docker/pulls/codercom/enterprise-clion?label=codercom%2Fenterprise-clion)](https://hub.docker.com/r/codercom/enterprise-clion)
```

**Review by bpmct — CHANGES_REQUESTED** [review]
I started requesting changes, but I realized the Dockerfile still downloads GoLand instead of CLion too, not just the docs. Is *there missing a commit @ntimo?

**Comment by ntimo** [comment]
@bpmct I fixed the commit sorry. Doing merge requests after waking up without Coffee is probably not that good of an idea :D But now it should be fine I guess.

**Review by bpmct — APPROVED** [review]

**Comment by jawnsy** [comment]
Looks good, thanks for this and sorry about the delay reviewing!

**Pull request merged**